resources/configurationFiles/analysisSNVCallingGRCh38.xml

```diff
@@ -0,0 +1,78 @@
+<configuration name='snvCallingAnalysisGRCh38' description='SNV Calling for NGS data.'
+               configurationType='analysis' class='de.dkfz.roddy.core.Analysis' workflowClass='de.dkfz.b080.co.snvpipeline.SNVCallingWorkflow'
+               runtimeServiceClass='de.dkfz.b080.co.common.BasicCOProjectsRuntimeService'
+               imports="commonCOWorkflowsSettings,snvCallingAnalysis"
+               canCreateTestdata='true'
+               listOfUsedTools="snvCalling,snvJoinVcfFiles,snvAnnotation,snvDeepAnnotation,snvFilter"
+               usedToolFolders="snvPipeline,tools"
+               cleanupScript="cleanupScript">
+
+  <configurationvalues>
+
+    <!--hg38 base directory-->
+    <cvalue name='hg38BaseDirectory' value='/omics/odcf/reference_data/legacy/ngs_share/assemblies/hg_GRCh38' type='path' />
+
+    <!-- Reference genomes -->
+    <cvalue name='REFERENCE_GENOME' value="${hg38BaseDirectory}/sequence/GRCh38_decoy_ebv_alt_hla_phiX.fa" type="path"/>
+    <cvalue name='INDEX_PREFIX' value="${hg38BaseDirectory}/indexes/bwa/bwa07_GRCh38_decoy_ebv_phiX_alt_hla_chr/GRCh38_decoy_ebv_alt_hla_phiX.fa" type="path"/>
+    <cvalue name="CHROM_SIZES_FILE" value="${hg38BaseDirectory}/stats/GRCh38_decoy_ebv_alt_hla_phiX.fa.chrLenOnlyACGT_realChromosomes.tsv" type="path" />
+    <cvalue name='CHROMOSOME_LENGTH_FILE' value='${hg38BaseDirectory}/stats/GRCh38_decoy_ebv_alt_hla_phiX.fa.chrLength.tsv' type="path" />
+
+    <!--Chr indices for the calling-->
```

Comment:
You don't have to use XML-comments. Use `<cvalue ... description="..."/>`.

For all the comments in here that are not e.g. comments on a whole group of configuration values.